_data/simple-icons.json

```diff
@@ -6437,7 +6441,7 @@
         {
             "title": "OpenVPN",
             "hex": "EA7E20",
-            "source": "https://openvpn.net/wp-content/themes/openvpn/assets/images/logo.svg",
+            "source": "https://commons.wikimedia.org/wiki/File:OpenVPN_logo.svg",
```

Comment:
Since a vector version of their logo is available on their website, it would be better to keep the source as https://openvpn.net/